src/tvshows.js

```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+
+const instance = axios.create({
+  baseURL: "https://api.themoviedb.org/3",
+  timeout: 1000,
+  headers: { Authorization: "Bearer " + import.meta.env.VITE_API_BEARER_TOKEN },
```

Comment:
Here we can use string interpolation.
```suggestion
  headers: { Authorization: `Bearer ${import.meta.env.VITE_API_BEARER_TOKEN}` },
```
A cheatsheet full of ES6 tricks like this: https://quickref.me/es6.html